test/service/storage/elasticsearch.test.js

```diff
@@ -3536,6 +3536,36 @@ describe('Test: ElasticSearch service', () => {
       elasticsearch._mExecute = sinon.stub().resolves(mExecuteResult);
     });
 
+    it('should call _mExecute with formated documents and source flag', () => {
+      const promise = elasticsearch.mCreateOrReplace(index, collection, documents, { source: false });
+
+      return promise
```

Comment:
I know that all unit tests are made like this here, but why not use async / await syntax ?

Reply:
I think you are right, I followed the standard of other unit tests.

Reply:
I agree with Olivier, new unit tests should be written using async/await syntax. The old ones will be update on fly